src/features/farming/shop/Plants.tsx

```diff
@@ -49,6 +52,11 @@ export const Plants: React.FC = () => {
     sell(1);
```

Comment:
the default param of `sell` is also set to `1`..

Reply:
I have removed the 1 to the "sell" function. The default param replace the empty value.